src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -314,3 +319,40 @@
                      priority   (get token-priority (:token token) 999)]
                  [(- fiat-value) priority]))
              calculated-tokens)))
+
+(defn on-asset-or-address-confirm
+  [{:keys [address address-prefix recipient collectible]}]
+  (let [[prefix]                    (when address (split-prefix-and-address address))
+        receiver-networks           (network-utils/resolve-receiver-networks
+                                     {:prefix           (or address-prefix prefix)
+                                      :testnet-enabled? (rf/sub [:profile/test-networks-enabled?])
+                                      :goerli-enabled?  (rf/sub [:profile/is-goerli-enabled?])})
```

Comment:
Aren't we getting a warning about subscribing outside of a reactive context with these?